features/bonding/examples/mco-bonding.md

```diff
@@ -0,0 +1,72 @@
+# Bonding using machine-config-operator (MCO)
+
+Project page [machine-config-operator](https://github.com/openshift/machine-config-operator)
+
+This document focusses on using `Machine Config operator (MCO)` for interface bonding and vlan configurations on a node. `MachineConfig` CRD is used to write configuration files onto the node. It expects the file contents to be supplied as encoded data per current ignition specifications, the encoding is done via `base64` in this document.
+
+### Sample bond interface file
+
+```
+BONDING_OPTS="miimon=140 mode=balance-rr"
+TYPE=Bond
+BONDING_MASTER=yes
+BOOTPROTO=dhcp
+NAME=bond1
+DEVICE=bond1
+ONBOOT=yes
+AUTOCONNECT_SLAVES=yes
```

Comment:
who are the slaves on this mode?

Reply:
Thanks. I've added another section to show sample slave interface config. Please review.